src/legacy/ui/public/new_platform/new_platform.ts

```diff
@@ -178,7 +178,8 @@ export const legacyAppRegister = (app: App<any>) => {
   legacyAppRegistered = true;
 
   require('ui/chrome').setRootController(app.id, ($scope: IScope, $element: JQLite) => {
-    const element = $element[0];
+    const element = document.createElement('div');
+    $element[0].appendChild(element);
```

Comment:
Just for the context, this is to reproduce

https://github.com/elastic/kibana/blob/4cb9bc44721e26ac2898eb11621b85f4418d913e/src/core/public/application/ui/app_container.tsx#L90

correct? Or is it related to the structure in `src/core/public/rendering/rendering_service.tsx` ?

Reply:
Yep that's right. Prior to this change, the app was being rendered into the `<div class="application">` directly.